### PR TITLE
Update 163.com.xml

### PR DESCRIPTION
**Pull request by ivysrono**

### CODE REVIEW

**Comment by jeremyn** [comment]
What's a URL where can I actually see the CORS problem? For example, https://dl.reg.163.com/gt?un=username looks fine to me.

Is removing the `securecookie` a required part of this fix?

**Comment by ivysrono** [comment]
@jeremyn Yes, we have to remove the securecookie to login.

**Comment by jeremyn** [comment]
On http://mail.163.com with the new ruleset, when I put text in the top and bottom text fields and click the blue button, I get a CAPTCHA check. I always fail the check and get this message in the Firefox console:

> Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://dl.reg.163.com/vfcp. (Reason: CORS header ‘Access-Control-Allow-Origin’ missing).

Do you get this? Do we need another `exclusion`?

**Comment by ivysrono** [comment]
@jeremyn Thank you very much!
I have update it.
Same issue: #7943

**Comment by jeremyn** [comment]
Is there a way to test the `<!-- get below tests after click the login button -->` URLs without a working username/password?

Can you please change the `<!--To fix the broken of email login. Test in mail.163.com / mail.126.com -->` comment [here](https://github.com/ivysrono/https-everywhere/blob/b80d4b99e3232f128b6e9c401b91704fbb925de0/src/chrome/content/rules/163.com.xml#L130) so that the problem domains are closer to the related exclusions? For example you can delete that line and edit [this line](https://github.com/ivysrono/https-everywhere/blob/b80d4b99e3232f128b6e9c401b91704fbb925de0/src/chrome/content/rules/163.com.xml#L135) to be something like:

```xml
		<!-- Avoid CORS problem when logging in at http://mail.163.com -->
```

and the same [here](https://github.com/ivysrono/https-everywhere/blob/b80d4b99e3232f128b6e9c401b91704fbb925de0/src/chrome/content/rules/163.com.xml#L149) but for http://mail.126.com. What do you think?

**Comment by ivysrono** [comment]
@jeremyn I'm afraid there is no way to test /gt and /| without a working username / password.
I have tested them lots times, it's ok now.

File: src/chrome/content/rules/163.com.xml [hunk 1]
```diff
@@ -133,22 +133,28 @@
...
-		<exclusion pattern="^http://passport\.126\.com/dl/(gt|ini|l)" />
-			<test url="http://passport.126.com/dl/gt?un=username" />
-			<!-- get below tests after click the login button -->
+		<exclusion pattern="^http://passport\.126\.com/dl/(ini|gt|l|cp|vfcp)" />
 			<test url="http://passport.126.com/dl/ini?pd=mail126&#x26;pkid=QdQXWEQ&#x26;pkht=mail.126.com" />
+			<!-- get below tests after click the login button -->
+			<test url="http://passport.126.com/dl/gt?un=username" />
 			<test url="http://passport.126.com/dl/l" />
+			<!-- get below tests while you enter wrong username or password -->
+			<test url="http://passport.126.com/dl/cp?pd=mail126&#x26;pkid=QdQXWEQ&#x26;random=6666666666666" />
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/163.com.xml]: Are there anything wrong with the CAPTCHA image?

**Comment by jeremyn** [comment]
We don't need the `/cp` tests specifically, do we? [Here](https://github.com/ivysrono/https-everywhere/blob/26602938378667260b1f00f5e4582695e4ec1545/src/chrome/content/rules/163.com.xml#L132) and [here](https://github.com/ivysrono/https-everywhere/blob/26602938378667260b1f00f5e4582695e4ec1545/src/chrome/content/rules/163.com.xml#L146). Can you delete them?

For removing the `securecookie`, do you want to replace it with a more specific `securecookie` with a `host` like `(analytics|fm|...)`? You don't have to, it's just an idea.

Please clarify in the `<!-- get below tests after click the login button -->`  comments that you mean with a working username/password, like `<!-- these tests happen after clicking the blue login button with a valid username/password -->`.

Also I can't test the `/gt` and `/l` `exclusion`s but I'll approve it anyway.

Other than that, this looks ready to go.

**Comment by ivysrono** [comment]
I don't know how to write securecookie rule.
Sorry.

**Comment by jeremyn** [comment]
Thanks, merged.

Don't worry about the `securecookie`, it's not a big deal. Maybe you and @gloomy-ghost can add one when you go back to pull request #7657.

**Pull request merged**